modules/fbx/fbx_parser/FBXMaterial.cpp

```diff
@@ -328,65 +328,58 @@ Video::Video(uint64_t id, const ElementPtr element, const Document &doc, const s
 	}
 
 	if (Content && !Content->Tokens().empty()) {
-		//this field is omitted when the embedded texture is already loaded, let's ignore if it's not found
```

Comment:
This seems to be straight from Assimp where it's still using this hack:
https://github.com/assimp/assimp/blob/556c89b5f143738720e17572faa8e272d4eec380/code/AssetLib/FBX/FBXMaterial.cpp#L304-L364

I don't see it intentionally throwing and catching exceptions, so I guess this code will just crash and instead of adding proper failure handling they just wrapped it in exceptions... *sigh*

So what can crash here? `token == nullptr` at least I suppose.

Reply:
I just changed it to replace the `try { } catch (...) { }` by `if (token)`, assuming that if it's failing it may return a nullptr. Add added another `ERR_FAIL_COND`, there was already one, it probably leaves things in a broken state if the condition is met but I'm not sure we even use the FBX `Video` class at all, so I don't want to do a deep dive.